content/hardware/04.pro/boards/portenta-x8/tutorials/09.image-flashing/content.md

```diff
@@ -76,7 +76,7 @@ On the Portenta Breakout, the DIP switches are identified by the label `BT_SEL`
 
 ![Breakout DIP switches](assets/breakout-dip-switches.png)
 
-You will need to connect one USB-C® end to the Portenta X8 and the other end (USB-C® or USB-A) to your computer. If the connection is established correctly, you will be able to see a newly connected device called `SE Blank M845S`.
+You will need to connect one USB-C® end to the Portenta X8 and the other end (USB-A®) to your computer, with a compatible hub in case of need. If the connection is established correctly, you will be able to see a newly connected device called `SE Blank M845S`.
```

Comment:
```suggestion
You will need to connect one USB-C® end to the Portenta X8 and the other end (USB-A) to your computer, with a compatible hub in case of need. If the connection is established correctly, you will be able to see a newly connected device called `SE Blank M845S`.
```